aiosmtpd/controller.py

```diff
@@ -207,7 +207,11 @@ def start(self):
                 # See comment about WSL1.0 in the _run() method
                 raise self._thread_exception
             else:
-                raise TimeoutError("SMTP server failed to start within allotted time")
+                raise TimeoutError(
+                    "SMTP server failed to start within allotted time. "
+                    "This might happen if the system is too busy. "
+                    "Try increasing the `ready_timeout` parameter."
+                )
```

Comment:
Much better now. The additional information should help users in fixing their problems.